### PR TITLE
[RORO-4906][EL] - add hmrcReportTechnicalIssueHelper to main template

### PR DESCRIPTION
**Pull request by elabeca**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -58,4 +58,6 @@ microservice {
 
 tracking-consent-frontend {
   gtm.container = "d"
-}
\ No newline at end of file
+}
+
+contact-frontend.serviceId = "Service Availability"
```
Comment (sureshhmrc) [inline conf/application.conf:63]: if it talks to `contact-frontend` then do we need its host/port config or not required ?

**Review by sureshhmrc — DISMISSED** [review]

File: conf/application.conf (hunk 1 above)
Comment (elabeca) [reply conf/application.conf:63]: there is a `contact-frontend.url` field which can be set to `http://localhost:9250` but that's already the default.

Comment (elabeca) [reply conf/application.conf:63]: I think I need to raise a PR in all environments though:
`microservice.services.contact-frontend.host: "www.qa.tax.service.gov.uk"`

Comment (sureshhmrc) [reply conf/application.conf:63]: hmm not sure, check their readme .. 
may be `play-frontend-hmrc` can do the url construction for us without us needing to add the url to each env

Comment (elabeca) [reply conf/application.conf:63]: spoke to plats-ui, they're saying it's all resolved internally https://github.com/hmrc/play-frontend-hmrc/blob/main/play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/helpers/HmrcReportTechnicalIssueHelper.scala.html#L29C19-L29C40

File: conf/application.conf [hunk 2]
```diff
@@ -60,4 +60,4 @@ tracking-consent-frontend {
   gtm.container = "d"
 }
 
-contact-frontend.serviceId = "Service Availability"
\ No newline at end of file
+contact-frontend.serviceId = "customs-service-status-frontend"
```
Comment (sureshhmrc) [inline conf/application.conf:63]: minor: i think you can specify `${appName}` rather than hardcoding the name again here

**Review by sureshhmrc — APPROVED** [review]

File: conf/application.conf (hunk 2 above)
Comment (elabeca) [reply conf/application.conf]: 🙏 @sureshhmrc

**Pull request merged**